HousingRepairsOnlineApi/Helpers/IRepairBookingResponseHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using HousingRepairsOnlineApi.Domain;
+using Microsoft.Azure.Cosmos;
+
+namespace HousingRepairsOnlineApi.Helpers
+{
+    public interface IRepairBookingResponseHelper
```

Comment:
The `Helper` suffix doesn't assist in understanding how it helps.
In this case, it's providing a mapping from `Repair` to one of the repair booking response classes so a more representative name would be `IRepairToRepairBookingResponseMapper`.